include/CMakeLists.txt

```diff
@@ -42,7 +42,7 @@ set( HEADERS_REGEX ".+\.(hpp|h|hxx|hh|h\\+\\+)$" )
 # to avoid flaky acrobatics with regex or glob expressions, copy main files directly
 install( FILES "graphblas.hpp" DESTINATION "${INCLUDE_INSTALL_DIR}" )
 set( root_files
-	"graphblas.hpp" "graphblas/backends.hpp" "graphblas/benchmark.hpp"
```

Comment:
Why is `graphblas.hpp` removed from here?

Reply:
The graphblas.hpp file is copied to the installation directory with the command in line 43. By including this file in line 44, it is copied in the subdirectory as well, which is incorrect.

Reply:
Got it, thanks!!